client/angular.json

```diff
@@ -46,7 +46,6 @@
               "optimization": true,
               "outputHashing": "all",
               "sourceMap": false,
-              "extractCss": true,
```

Comment:
Is this no longer a feature in ng11? Extracting the CSS can be nice because CSS can be parsed in parallel w/ the javascript, whereas, with embedded CSS, the javascript needs parsed and executed before the CSS is inserted into the `<head>`

Reply:
this was removed by default when running `ng update`.
![image](https://user-images.githubusercontent.com/11415127/108753117-51f7fd80-7512-11eb-9f3a-b67581cb0492.png)